conda_forge_tick/deploy.py

```diff
@@ -14,9 +15,9 @@ def _run_git_cmd(cmd):
     return subprocess.run(cmd, shell=True, check=True)
 
 
-def deploy(dry_run=False):
-    """Deploy the graph to github"""
-    if dry_run:
+def deploy(ctx: CliContext = CliContext()):
```

Comment:
Just commenting on this instance here: Should we provide a default argument here? This makes it easy to "forget" to pass the context and this function is just invoked once with a CliContext anyway

Reply:
Makes sense, and also this makes no sense anymore with the new central CLI entry point. Fixed in e97ab2d7